src/helperFunctions/getUser.ts

```diff
@@ -1,13 +1,9 @@
-import { USER_SELF } from '@/constants/url';
-import fetch from '@/helperFunctions/fetch';
+import useUserData from '@/hooks/useUserData';
 
 const userData = async () => {
+    const { data } = useUserData();
     try {
-        const { requestPromise } = fetch({
-            url: USER_SELF,
-        });
-        const response = await requestPromise;
-        return response.data;
+        return data;
     } catch (error) {
         console.error(error);
         return error;
```

Comment:
if we have removed the fetch() then do we need this file, can't we use it directly useUserData where ever we are calling userData()

Reply:
Oh yes! This file has become redundant actually. Removed!

Reply:
Yes! It had become redundant . Removed!